### PR TITLE
[KO] Update translation

### PR DESCRIPTION
**Pull request by marocchino**
@Dalgona, @malkoG, @devleoper, @riseshia review please.

I use this command for diff:

```bash
git diff  ac2ad1d..65e4a61 --word-diff -- lessons/
```

And still remain two docs:
- lessons/basics/iex_helpers.md
- lessons/specifics/debugging.md 

I wanted to keep the previous translations. but, there may be new translations of previously translated parts due to the many diff.

### CODE REVIEW

**Comment by marocchino** [comment]
Oh, and if possible, I want you to translate the two remaining documents. Who has time?

**Comment by devleoper** [comment]
People got busier these days. 😣 I'll look into and translate **IEx Helpers** lesson. 🙌 

@Suckzoo Can I ask you to review this? You have keen eyes so I would like to ask you! 👀

File: ko/lessons/specifics/plug.md [hunk 1]
```diff
@@ -51,17 +58,117 @@ defmodule HelloWorldPlug do
...
+
+  plug :match
+  plug :dispatch
+
+  get "/", do: send_resp(conn, 200, "Welcome")
+  match _, do: send_resp(conn, 404, "Oops!")
+end
+```
+
+이것은 최소한의 라우터이지만 코드는 꽤 자명합니다.
```
Comment (Suckzoo) [inline ko/lessons/specifics/plug.md]: I suggest this sentence, as introducing the pronoun `이것` may make the sentence awkward:
"위의 코드는 최소한으로 작성된 라우터지만, 무슨 의미로 작성되었는지 쉽게 알 수 있습니다."

Comment (marocchino) [inline ko/lessons/specifics/plug.md]: Too verbose.

Comment (Suckzoo) [inline ko/lessons/specifics/plug.md]: OK. Nevermind :)

**Pull request merged**